src/azure-cli/azure/cli/command_modules/appconfig/_utils.py

```diff
@@ -34,19 +34,51 @@ def construct_connection_string(cmd, config_store_name):
 
 
 def resolve_store_metadata(cmd, config_store_name):
+    resource_group = None
+    endpoint = None
     try:
         config_store_client = cf_configstore(cmd.cli_ctx)
         all_stores = config_store_client.list()
         for store in all_stores:
             if store.name.lower() == config_store_name.lower():
-                # Id has a fixed structure /subscriptions/subscriptionName/resourceGroups/groupName/providers/providerName/configurationStores/storeName"
-                return store.id.split('/')[4], store.endpoint
+                if resource_group is not None:
+                    raise CLIError('Multiple configuration stores found with name {}.'.format(config_store_name))
+                else:
```

Comment:
Unnecessary "else" after "raise" (no-else-raise)
```suggestion
                if resource_group is None:
```